app/controllers/users_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
       flash[:success] = "Welcome to youBlog #{@user.username}"
       redirect_to user_path(@user)
     else
-      flash[:danger] = "Sorry, email already exists!"
+      flash[:danger] = "#{@user.errors.full_messages[1]}"
```

Comment:
You might be able to use `.to_sentence` here if there's always going to be one error for signup issues

`#{@user.errors.full_messages[1]}.to_sentence`